examples/source_examples/veeva_vault_objects/connector.py

```diff
@@ -0,0 +1,243 @@
+# This is a simple example for how to work with the fivetran_connector_sdk module.
+# This code is currently configured to Retrieve Details from All Object Types in Veeva Vault and then utilize VQL to retrieve all Object records, creating one table per object
+# You will need to provide your own Veeva Vault credentials for this to work --> subdomain, username, and password variables in configuration.json 
+# Retrieve Details from All Object Types endpoint: https://developer.veevavault.com/api/24.2/#retrieve-details-from-all-object-types
+# VQL endpoint: https://developer.veevavault.com/api/24.2/#vault-query-language-vql
+# You can also add code to extract from other endpoints as needed.
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+# Import required classes from fivetran_connector_sdk
+
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests
+import json
+import datetime
+
+# Function to start a Veeva Session, inputs needed are: username, password, and subdomain -- all from the configuration.json file
+# Auth endpoint: https://developer.veevavault.com/api/24.2/#user-name-and-password 
+# The startVeevaSession function takes 1 parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+
+def startVeevaSession(configuration: dict):
+    username = configuration.get('username')
+    password = configuration.get('password')
+    auth_url = f"https://{configuration.get('subdomain')}/api/v24.2/auth"
```

Comment:
we are calling the field as sub-domain but then taking the entire value in credentials as the URL

it should be 
```auth_url = f"https://{configuration.get('subdomain')}.veevavault.com/api/v24.2/auth"```